.github/workflows/functional-tests-workflow.yml

```diff
@@ -35,10 +35,10 @@ jobs:
         with:
           ref: ${{ inputs.checkout-ref }}
           repository: ${{ inputs.checkout-repository }}
-      - name: Set up Python 3.10
+      - name: Set up Python 3.11
```

Comment:
BTW why do we use 3.11 for functional tests instead of 3.8 ? I believe using 3.8 may help us to avoid issues like this https://github.com/dbt-athena/dbt-athena/issues/483. Did I miss smth ?

Reply:
3.11 is slightly faster, so I put 3.11 only for that. 
But I'm happy to switch to 3.8, to cover edge cases.